plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeTableWithCustomLocation.java

```diff
@@ -63,18 +60,18 @@ public void testCreateAndDrop()
         String tableName = "test_create_and_drop" + randomNameSuffix();
         assertQuerySucceeds(format("CREATE TABLE %s AS SELECT 1 as val", tableName));
         Table table = metastore.getTable(SCHEMA, tableName).orElseThrow();
-        assertThat(table.getTableType()).isEqualTo(TableType.MANAGED_TABLE.name());
+        assertThat(table.getTableType()).isEqualTo(MANAGED_TABLE.name());
```

Comment:
nit: This change is unrelated to the file system migration. Ideally, it should be a separated commit.